### ISSUE TITLE
iOS Accessibility - VoiceOver rotor functions are unsupported.

### ISSUE THREAD

**Issue by SergeantConfused** [bitwarden/android#2589]
### Steps To Reproduce

1. Activate VoiceOver on iOS.
2. Open Bitwarden and create a new vault item.
3. Use Rotor Functions (https://support.apple.com/en-us/HT204783) set to Words.

### Expected Result

VoiceOver should navigate through the words in the form.

### Actual Result

VoiceOver does not read any words from the form.

### Screenshots or Videos

N/A.

### Additional Context

It also does not work when Rotor functions are set to (Characters).

### Operating System

iOS

### Operating System Version

16.5.1

### Device

iPhone SE (2020)

### Build Version

2023.5.0

### Beta

- [ ] Using a pre-release version of the application.

**Comment by Greenderella** [comment]
If anybody else wishes to add any further information/screenshots/recordings etc., please feel free to do it here - Bitwarden's engineering team will be happy to review these.

**Comment by vvolkgang** [comment]
Issue migrated to https://github.com/bitwarden/mobile/issues/2589
